bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala

```diff
@@ -0,0 +1,100 @@
+package edu.gemini.spModel.gemini.ghost
+
+import java.beans.PropertyDescriptor
+import java.util.{Collections, List => JList, Map => JMap, Set => JSet}
+
+import edu.gemini.pot.sp.{ISPObservation, SPComponentType}
+import edu.gemini.shared.util.immutable.ImList
+import edu.gemini.shared.util.immutable.ScalaConverters._
+import edu.gemini.spModel.core.Site
+import edu.gemini.spModel.data.ISPDataObject
+import edu.gemini.spModel.data.config.{DefaultParameter, DefaultSysConfig, ISysConfig, StringParameter}
+import edu.gemini.spModel.data.property.{PropertyProvider, PropertySupport}
+import edu.gemini.spModel.obscomp.{InstConfigInfo, InstConstants, SPInstObsComp}
+import edu.gemini.spModel.pio.{ParamSet, PioFactory}
+import edu.gemini.spModel.seqcomp.SeqConfigNames
+import edu.gemini.spModel.target.env.AsterismType
+
+import scala.collection.immutable.TreeMap
+import scala.collection.JavaConverters._
+
+
+/** The GHOST instrument SP model.
+  * Note that we do not override clone since private variables are immutable.
+  */
+class Ghost extends SPInstObsComp(Ghost.SP_TYPE) with PropertyProvider {
```

Comment:
Let's go ahead and `final class` it.  What do you mean "since private variables are immutable"?

Reply:
I guess it doesn't really mean anything, since right now, the class contains no variables: the only value (posangle) is handled by the superclass.